packages/rehype-postfix-footnote-anchors/__tests__/__snapshots__/index.js.snap

```diff
@@ -5,18 +5,9 @@ exports[`{"gfm":false,"commonmark":false,"footnotes":true} footnote-split 1`] =
 <div class=\\"footnotes\\">
 <hr>
 <ol>
-<li id=\\"fn-first-postfix\\">
-<p>first def</p>
-<a href=\\"#fnref-first-postfix\\" class=\\"footnote-backref\\">↩</a>
-</li>
-<li id=\\"fn-third-postfix\\">
-<p>third def</p>
-<a href=\\"#fnref-third-postfix\\" class=\\"footnote-backref\\">↩</a>
-</li>
-<li id=\\"fn-1-postfix\\">
-<p>second def</p>
-<a href=\\"#fnref-1-postfix\\" class=\\"footnote-backref\\">↩</a>
-</li>
+<li id=\\"fn-first-postfix\\">first def<a href=\\"#fnref-first-postfix\\" class=\\"footnote-backref\\">↩</a></li>
+<li id=\\"fn-1-postfix\\">second def<a href=\\"#fnref-1-postfix\\" class=\\"footnote-backref\\">↩</a></li>
+<li id=\\"fn-third-postfix\\">third def<a href=\\"#fnref-third-postfix\\" class=\\"footnote-backref\\">↩</a></li>
```

Comment:
Corresponding fixture:

```md
a[^first]b[^second def]c[^third]

[^first]: first def
[^third]: third def
```

The behavior changed, probably for the better.

Reply:
I don't know what you think changed, but from what I can see, only the order of the three footnotes changed.

Reply:
Yeah I was talking about the order in which footnotes are rendered. Footnotes are displayed in the same order as their definition.

```md
a[^first_footnote_reference]

b[^`b` second footnote reference but first footnote definition]

c[^last_footnote_reference]

[^last_footnote_reference]: `c` last footnote reference but second footnote definition
[^first_footnote_reference]: `a` first footnote reference but last footnote definition
```

Before:

![image](https://user-images.githubusercontent.com/2022803/73589529-7207e980-44d7-11ea-8c67-cd8a20d961fe.png)

After:

![image](https://user-images.githubusercontent.com/2022803/73589535-7cc27e80-44d7-11ea-90e0-3e0e0dbac87c.png)

In the HTML ordered list, the list items 1/2/3 don't match the footnote references numbers.

Reply:
Ok, I see what you mean, added it to the issue status to get an approval of the change.

Reply:
this cannot be good : if you click on "3" you get redirected to "c" whereas it should lead you to "a"

Reply:
Yeah, the footnote reference (superscript) and the list item number should match.

a^3, 3. a

Reply:
This is fixed in zmarkdown; the behaviour is normal in rehype, so the test will not be updated for the package.